app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -94,18 +109,6 @@ const StashIcon: OcticonSymbolVariant = {
 
 const GitIgnoreFileName = '.gitignore'
 
-function getCheckBoxValueFromIncludeAll(includeAll: boolean | null) {
```

Comment:
Mentioned elsewhere, I would bother extracting this for the sake of testing. Only worth extracting if we were to reuse it elsewhere.